server/tests/db-entry-mockdata.js

```diff
@@ -0,0 +1,23 @@
+const entries = {
```

Comment:
Do we still require this, if yes for what?

Reply:
yes, am using them for testing